templates/manifest.json

```diff
@@ -17,6 +17,10 @@
             },
             "aliases": [
                 "getting_started_python"
+            ],
+            "showcaseFiles": [
+                "src/main.py",
+                "src/__main__.py"
```

Comment:
For the Python ones, I would not show the `src/__main__.py` file, the logic there isn't really that necessary to show off.

Reply:
I guess we sort the files by the order in the array right? Even though I don't know much about Python, it would be weird for me to find that some files are seemingly missing from the template. Or, the other way around, that some files were added to my actor even though they were not in the list.

I kinda understand that for supporting files, like eslint.json or something like that. But I think we should always show all the source files. Would be weird to e.g. find an import of a file that's not visible in the template.